### PR TITLE
add option createLinkIcon: able to use another icon library

### PR DESCRIPTION
**Pull request by Sysix**
If you would like to use another library or customize your library via fontello.

### CODE REVIEW

File: src/js/shariff.js [hunk 1]
```diff
@@ -112,6 +112,12 @@ Shariff.prototype = {
 
         flattrUser: null,
 
+        createLinkIcon: function($shareLink, service) {
+            if (typeof service.faName !== 'undefined') {
+                $shareLink.prepend('<span class="fa ' +  service.faName + '">');
```
Comment (core23) [inline src/js/shariff.js]: You should use `<i class="fa ' +  service.faName + '" aria-hidden="true"></i>` instead of the old span tag

Comment (Sysix) [inline src/js/shariff.js]: I copied the code from the source. But your right, I can change it if you would like ;)

Comment (liayn) [inline src/js/shariff.js]: keep in mind that the `<i>` tag may cause problems on some pages when used. a neutral `<span>` is indeed less intrusive

Comment (core23) [inline src/js/shariff.js]: The `<i>` is recommended for font awesome, also the `<span>` tag *is more semantically correct* 😕 
http://fontawesome.io/examples/

Comment (core23) [inline src/js/shariff.js]: At least we should add the `aria-hidden` attribute.

Comment (Sysix) [inline src/js/shariff.js]: added

**Comment by Sysix** [comment]
can someone please update the docs?

**Comment by compeak** [comment]
Looks like this is dead.

**Pull request closed without merging**